net/http/partitioned_host_state_map_unittest.cc

```diff
@@ -16,8 +16,8 @@ namespace net {
 
 namespace {
 
-using PartitionedMap = PartitionedHostStateMap<
-    std::map<std::array<uint8_t, crypto::kSHA256Length>, std::string>>;
+using HostHash = std::array<uint8_t, crypto::kSHA256Length>;
+using PartitionedMap = PartitionedHostStateMap<std::map<HostHash, std::string>>;
 
 std::array<uint8_t, crypto::kSHA256Length> HashHost(
```

Comment:
`HostHash HashHost(...`

Reply:
Fixed in eb96a2aa698459aacbf30e3e2196b95e0093d33b.